build.rs

```diff
@@ -4,6 +4,9 @@ use std::path::PathBuf;
 
 fn main() {
     println!("cargo:rerun-if-changed=tests");
+    if cfg!(not(test)) {
+        return;
+    }
```

Comment:
when you create `code.php` alone, and try to run the binary, the build script will fail as it won't be able to build the tests, hence why i added this, now it will allow the binaries to run, and only fail if we are build for tests.